wizeline/project-lab
# Feat/allow admins to add/remove new admins - #211

#### What does this PR do?

Allows users with `ADMIN` role to add new admins or remove admin role from current admins

#### Where should the reviewer start?

The reviewer show log in as an Admin and go to the Manager page

#### How should this be manually tested?

1. Pull the changes and run the project
2. Log in with an Admin user
3. Go to the Manager page
4. Check that there are now 2 tabs (Filter Tags and Admin users)
5. In the Admin users tab, click on `ADD NEW ADMIN` and enter an email. Validate that the email is now listed in the Admins chart, also validate that the user now exists in the `User` table from the db.
6. Click on the trash can button for any of the users listed in the chart and confirm the Admin role removal. Check that the user now has the `USER` role assigned in the `User` table in the db and the user is not listed in the chart anymore.

#### What are the relevant tickets?

#211 

#### Screenshots

<img width="1026" alt="image" src="https://user-images.githubusercontent.com/96010361/155794673-60f15898-801a-4ec9-8727-3924f487d53f.png">

<img width="1026" alt="image" src="https://user-images.githubusercontent.com/96010361/155794875-1280be81-d650-4d83-8f27-e077c98f355f.png">
